cura/Settings/ExtruderManager.py

```diff
@@ -13,9 +13,13 @@
 #
 #   This keeps a list of extruder stacks for each machine.
 class ExtruderManager(QObject):
-    ##  Signal to notify other components when the list of extruders changes.
+    ##  Signal to notify other components when the list of extruders for a machine definition changes.
     extrudersChanged = pyqtSignal(QVariant)
 
+    ## Signal to notify other components when the global container stack is switched to a definition
+    #  that has different extruders than the previous global container stack
+    globalContainerStackDefinition = pyqtSignal()
```

Comment:
I prefer to call signals that trigger when something changes "somethingChanged", so in this case "extruderDefinitionsChanged" or something similar.